src/qibocal/protocols/characterization/flux_dependence/avoided_crossing.py

```diff
@@ -177,106 +179,109 @@ def _plot(data: AvoidedCrossingData, fit: AvoidedCrossingResults, qubit):
             row=1,
             col=i + 1,
         )
+        if fit:
```

Comment:
Maybe
```suggestion
        if fit is not None:
```
?

(and in case, the parameter's type should be updated with `Optional[...]`)